src/components/Icon.ts

```diff
@@ -1,17 +1,28 @@
-import { h } from 'vue'
+import { h, inject, reactive } from 'vue'
 
-import icons from '../partials/icons'
-import { Data } from '../types'
+import icons, { IconName } from '../partials/icons'
+import { CarouselConfig, Data } from '../types'
+import { defaultConfigs } from '@/partials/defaults'
+
+function isIconName(candidate: string): candidate is IconName {
+  return candidate in IconName
+}
 
 const Icon = (props: Data) => {
+  const config: CarouselConfig = inject('config', reactive({ ...defaultConfigs }))
   const iconName = props.name
-  if (!iconName || typeof iconName !== 'string') {
+  if (!iconName || typeof iconName !== 'string' || !isIconName(iconName)) {
     return
   }
+
   const path = icons[iconName]
   const pathEl = h('path', { d: path })
 
-  const iconTitle = (props.title || iconName) as string
+  const iconTitle = (props.title
+    || (config.labels?.iconAriaLabels ? config!.labels!.iconAriaLabels![iconName]! as string : undefined)
+    || iconName
+  ) as string
```

Comment:
Still uses the icon name as a fallback but prefers the configured title. Not sure if that's alright the way it is or if we should inject the translated aria label via props instead.